android/app/src/main/jni/string_script.inc

```diff
@@ -0,0 +1,66 @@
+/* A sample string script. */
+
+const char *string_script = R"(
```

Comment:
Actually, I would suggest that the .inc file be *only* the raw string that you would load from a file.
That is, the main.cpp would have

```
conat char* string_script =
#include "string_script.inc"
;
```
(I would have wanted the #include inside the raw string, but then I think the preprocessor would never see the directive!)